ui-tests/src/test/java/io/syndesis/qe/steps/customizations/api_client_connectors/ApiClientConnectorsSteps.java

```diff
@@ -0,0 +1,180 @@
+		for (List<String> row : dataRows) {
+			sourceTypes.add(row.get(0));
+			urls.add(row.get(1));
+		}
+
+		uploadSwaggerWizardStep.upload(sourceTypes.get(0), urls.get(0));
+	}
+
+	@Then("^(\\w+) reviews swagger actions$")
+	public void she_reviews_swagger_actions(String user) throws Throwable {
+		reviewSwaggerActionsWizardStep.validate();
+		reviewSwaggerActionsWizardStep.review();
+		reviewSwaggerActionsWizardStep.nextWizardStep();
+	}
+
+	@Then("(\\w+) sets up security by \"([^\"]*)\"$")
+	public void she_sets_up_security_by(String user, String authType) throws Throwable {
+		securityWizardStep.validate();
+		//String authType, String authenticationUrl, String authorizationUrl, String scopes
+		switch (authType) {
+			case "OAuth 2.0":
+				securityWizardStep.setUpSecurity(TestConfiguration.syndesisUrl() + TestConfiguration.syndesisCallbackUrlSuffix());
+				break;
+			default:
+				throw new UnsupportedOperationException("The Auth type < " + authType + "> is not implemented by the test.");
+		}
+	}
+
+	@Then("^(\\w+) reviews general info$")
+	public void she_reviews_general_info(String user) throws Throwable {
+		generalConnectorInfoWizardStep.validate();
+		generalConnectorInfoWizardStep.review();
+		generalConnectorInfoWizardStep.validate();
+	}
+
+	@Then("(\\w+) is presented with the new connector \"([^\"]*)\"$")
+	public void she_is_presented_with_the_new_connector(String user, String connectorName) throws Throwable {
+		Assert.assertTrue("Connector [" + connectorName + "] is not listed.", apiClientConnectorsPage.isConnectorPresent(connectorName));
+	}
+
+	@Then("^(\\w+) creates new API connector \"([^\"]*)\"$")
+	public void she_creates_new_api_connector(String user, String connectorName, DataTable properties) throws Throwable {
+
+		String securityAuthType = null;
+
+		for (List<String> property : properties.raw()) {
+			switch (property.get(0)) {
+				case "security":
+					switch (property.get(1)) {
+						//| security | authType | OAuth 2.0 |
+						case "authType":
+							securityAuthType = property.get(2);
+							break;
+						default:
+					}
+					break;
+
+				default:
+			}
+		}
+
+		CommonSteps commonSteps = new CommonSteps();
+		commonSteps.navigateTo(user, "Customizations");
+		commonSteps.validatePage(user, "Customizations");
+		commonSteps.clickOnLink("API Client Connectors");
+		commonSteps.validatePage(user, "API Client Connectors");
+		she_opens_new_api_connector_wizard(user);
+
+		List<String> row = asList("file", "swagger_files/api_client_connector/petstore_swagger.json");
+		List<List<String>> dataTable = new ArrayList<>();
+		dataTable.add(row);
+
+		she_uploads_swagger_file(user, DataTable.create(dataTable));
+		she_navigates_to_the_next_Api_Connector_wizard_step(user, "Review Swagger Action");
+		she_navigates_to_the_next_Api_Connector_wizard_step(user, "Security");
+
+		List<String> secRow = asList("authType", "authenticationUrl", "https://syndesis.192.168.42.188.xip.io/api/v1/credentials/callback", "scopes");
```

Comment:
Hardcoded URL won't work across CI jobs.

Reply:
deleted. It was residual unused one